components/dashboard/src/user-settings/SSHKeys.tsx

```diff
@@ -69,15 +69,12 @@ export function AddSSHKeyModal(props: AddModalProps) {
                     </Alert>
                 )}
                 <div className="text-gray-500 dark:text-gray-400 text-md">
-                    Add an SSH key for secure access workspaces via SSH.{" "}
-                    <a href="/docs/configure/ssh" target="gitpod-ssh-doc" className="gp-link">
-                        Learn more
-                    </a>
+                    Add an SSH key for secure access workspaces via SSH.
```

Comment:
the sentence sounds odd as id. how about adding a "to"?

`Add an SSH key for secure access to workspaces via SSH.`

Reply:
Nice catch! ⚾  
Updated in 807f1ac. 🤖